pulsar-broker/src/test/java/org/apache/pulsar/client/impl/PatternTopicsConsumerImplTest.java

```diff
@@ -469,11 +469,6 @@ public void testStartEmptyPatternConsumer() throws Exception {
             .messageRoutingMode(org.apache.pulsar.client.api.MessageRoutingMode.RoundRobinPartition)
             .create();
 
-        List<String> topicNames = Lists.newArrayList(topicName1, topicName2, topicName3);
-        NamespaceService nss = pulsar.getNamespaceService();
-        doReturn(CompletableFuture.completedFuture(topicNames)).when(nss)
-                .getListOfPersistentTopics(NamespaceName.get("my-property/my-ns"));
```

Comment:
Why did you remove these three lines?

Reply:
This is a PR added before trying to fix that test, it didn't work.

https://github.com/apache/pulsar/pull/16599/files